.github/workflows/node.js.yml

```diff
@@ -4,28 +4,29 @@
 name: Node.js CI
 
 on:
-  push:
```

Comment:
Is this because of the linter? If so it can probably ignore config files

Reply:
reverted it and added to ignore